sandbox/Varsha/aks/aks-varsha.tf

```
@@ -0,0 +1,39 @@
+resource "azurerm_kubernetes_cluster" "varsha-aks" {
+ lifecycle {
+    prevent_destroy = true
+    ignore_changes = [microsoft_defender]
+  }
+
+  name                = "varsha-aks"
+  location            = azurerm_resource_group.varsha-rs.location
+  resource_group_name = azurerm_resource_group.varsha-rs.name
+  dns_prefix          = "LRS"
+  enable_pod_security_policy       = false
+  private_cluster_enabled          = false
+  azure_policy_enabled             = true
+  http_application_routing_enabled = false # Instead of this, Nginx ingress controller is used to route private http traffic in AKS
+
+
+  default_node_pool {
+    name       = "varsha-test"
+    node_count = 2
+    vm_size    = "Standard_D2_v2"
+    type                 = "VirtualMachineScaleSets"
+    max_pods             = 2
```

2 is very less.. kubernetes itself will try to spin more that 2 system services 
```suggestion
    max_pods             = 50
```

---

incorporated